acs/acs.go

```diff
@@ -0,0 +1,37 @@
+package acs
+
+type ACS interface {
+	Receive(batch []byte)
+	sendConsensus()
+	sendBatch(id int)
+	sendBinary(id int)
+}
+
+type memACS struct {
+	// TODO : ACS must have rbc instances map
+	// TODO : ACS must have bba instances map
+}
+
+func NewACS() ACS {
+	return &memACS{}
+}
+
+// Receive handles batch which is send from honeybadger.
+func (m *memACS) Receive(batch []byte) {
+
+}
+
+// send sends consensus batch to honeybadger.
+func (m *memACS) sendConsensus() {
```

Comment:
I think sendConsensus look like that 'now, we start the consensus'. sending batch to honeybadger is result of consensus.

and maybe you had a mistake writing method name and comment. `send`